bindings/profilers/wall.hh

```diff
@@ -65,7 +65,11 @@ class WallProfiler : public Nan::ObjectWrap {
   bool started_ = false;
   bool workaroundV8Bug_;
   bool detectV8Bug_;
+  bool collectCpuTime_;
+  bool isMainThread_;
   int v8ProfilerStuckEventLoopDetected_ = 0;
+  int64_t startThreadCpuTime_ = 0;
+  int64_t startProcessCpuTime_ = 0;
```

Comment:
`startProcessCpuTime_` looks unused (write-only access) but I presume you'll be using it later.